index.bs

```diff
@@ -844,7 +844,7 @@ in either a RST_STREAM frame or a STOP_SENDING frame).
 
 <pre class="idl">
 dictionary StreamAbortInfo {
-  unsigned long long errorCode = 0;
+  octet errorCode = 0;
```

Comment:
I think we should probably use [EnforceRange] here. The default modulo behaviour would be confusing, and [Clamp] would probably be unhelpful.